build/yamls/kafka-flow-collector.yaml

```diff
@@ -77,7 +77,7 @@ spec:
     spec:
       containers:
         - name: kafka
-          image: harbor-repo.vmware.com/dockerhub-proxy-cache/confluentinc/cp-kafka:latest
+          image: antrea/confluentinc-kafka:v6.2.0
```

Comment:
Let's use the following VMware registry with no rate limiting like other images:
projects.registry.vmware.com/antrea

Reply:
Updated. Thanks!